philippika.csl

```diff
@@ -0,0 +1,257 @@
+  <info>
+    <title>Philippika (German)</title>
+    <id>http://www.zotero.org/styles/philippika</id>
+    <link href="http://www.zotero.org/styles/philippika" rel="self"/>
+    <link href="http://www.harrassowitz-verlag.de/category_281.ahtml" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/cahiers-du-centre-gustave-glotz" rel="template"/>
+    <author>
+      <name>Nicolas Monteix</name>
+      <email>nicolas.monteix@univ-rouen.fr</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="history"/>
+    <category field="humanities"/>
+    <issn>1613-5628</issn>
+    <summary>Citation style for the Philippika collection (Harrassowitz Verlag). Language normally set as German. Using Zotero, to set English, use the Quick Locale Switcher (see https://www.zotero.org/support/supported_languages)</summary>
+    <updated>2015-03-06T23:37:43+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="de"/>
+  <macro name="author">
+    <names variable="author" delimiter=", ">
+      <name delimiter-precedes-last="never" name-as-sort-order="all" and="symbol" sort-separator=", " initialize-with="." delimiter=", " et-al-min="3" et-al-use-first="1"/>
+      <et-al/>
+      <substitute>
+        <text macro="editor"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor" delimiter=", ">
+      <name delimiter-precedes-last="never" name-as-sort-order="all" and="symbol" sort-separator=", " initialize-with="." delimiter=", " et-al-min="3" et-al-use-first="1"/>
+      <et-al/>
+      <label form="short" prefix=" (" text-case="lowercase" suffix=")"/>
+      <substitute>
+        <names variable="container-author" delimiter=", ">
+          <name delimiter-precedes-last="never" name-as-sort-order="all" and="symbol" sort-separator=", " initialize-with="." delimiter=", " et-al-min="3" et-al-use-first="1"/>
+        </names>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="volume">
+    <group>
+      <number variable="volume" form="numeric"/>
+    </group>
+  </macro>
+  <macro name="pages">
+    <text variable="page"/>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name delimiter-precedes-last="never" name-as-sort-order="all" and="symbol" form="short" delimiter=", " et-al-min="3" et-al-use-first="1"/>
+      <et-al font-variant="normal"/>
+      <substitute>
+        <names variable="editor"/>
+        <text variable="title-short"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="publisher-date">
+    <group>
+      <text macro="publisher" prefix=", " suffix=" "/>
+      <text macro="edition"/>
+      <text macro="year-date"/>
+    </group>
+  </macro>
+  <macro name="access">
+    <group>
+      <text term="accessed" suffix=" "/>
```

Comment:
in German that gives you e.g. 7 März 2015 -- that seems wrong. You probably want to use a localized date here, like ```<date variable="accessed" form="numeric"/>``` and then no date parts.